NVorbis/Contracts/IContainerReader.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NVorbis.Contracts
+{
+    /// <summary>
+    /// Provides an interface for a Vorbis logical stream container.
+    /// </summary>
+    public interface IContainerReader : IDisposable
+    {
+        /// <summary>
+        /// Gets or sets the callback to invoke when a new stream is encountered in the container.
+        /// </summary>
+        Func<IPacketProvider, bool> NewStreamCallback { get; set; }
```

Comment:
This is a small thing but I noticed you had the delegate definition GetPacketGranuleCount in IPacketProvider. Any reason not to do that here or use Func instead of GetPacketGranuleCount?

Reply:
Good point.  I'll change this one to use a declared delegate instead.